tests/components/hue/test_sensor_base.py

```diff
@@ -473,7 +477,9 @@ async def test_hue_events(hass, mock_bridge):
     mock_bridge.mock_sensor_responses.append(new_sensor_response)
 
     # Force updates to run again
-    await mock_bridge.sensor_manager.coordinator.async_refresh()
+    async_fire_time_changed(
+        hass, dt_util.utcnow() + sensor_base.SensorManager.SCAN_INTERVAL
```

Comment:
Do we need to increase time since last refresh to have the coordinator do the update? Now we're just firing the same time again every time.

Reply:
Since we don't patch `dt_util.utcnow`, the coordinator will schedule a refresh again at now + scan_interval, so we can fire the same time.